requirements.txt

```diff
@@ -0,0 +1,22 @@
+# some of these are for testing only
```

Comment:
I ran a pip freeze in my virtualenv to get these, the nice thing about having this file is that wherever you are if you need to get the requirements in place you can just run pip install -r path/to/this/requirements.txt

Reply:
should just need these:
 pip install unittest2 nose2 nose2-gae WebTest

Reply:
see https://cloud.google.com/appengine/docs/standard/python/tools/using-libraries-python-27

Reply:
Ack I'll work on another pass with a test runner that leverages the appengine builtins and trim this list down as part. Thanks for walking me through this!